src/aics-image-viewer/components/AxisClipSliders/index.tsx

```diff
@@ -176,7 +177,7 @@ export default class AxisClipSliders extends React.Component<AxisClipSlidersProp
         const thicknessReduce = isActiveAxis ? step - stepEpsilon : 0.0;
 
         const start = values[0] / max - 0.5;
-        const end = values[1] / max - 0.5 - thicknessReduce / max;
+        const end = values[values.length - 1] / max - 0.5 - thicknessReduce / max;
```

Comment:
i might add a comment here to say that this is meant to handle both cases when values is of length 1 or 2

Reply:
```// values will be of length 1 when slider has a single handle, and of length 2 when slider has 2 handles. ```

Reply:
there will probably be a future modification to this code to handle true slice thickness (i.e. a single slice should have nonzero max-min) as we discussed in slack thread, and maybe treat thicknessReduce as some kind of minimum clamping if the max-min thickness is too small?  half baked idea but we want to make this as accurate as possible for single-slice viewing

Reply:
Right - I'm thinking I'd like to make one more PR to roi-slider-updates after all issues on the tracker are resolved to clarify this chunk of code, i.e. how we translate slices to volume-viewer clipping coordinates, and to address other feedback from this last round of meetings and slack conversation.